utils/translation-gpt/index.mjs

```diff
@@ -0,0 +1,120 @@
+import {readFileSync, writeFileSync} from 'fs';
+import {env} from 'process';
+
+const fileToTranslate = JSON.parse(
```

Comment:
For `.mjs` file, to import JSON you have to add `assert {type: 'json'}` at the end of the import line.

And the `assert` keyword was tripping up es-lint/prettier.
So I used a wonky import instead.